src/shadertools/lib/platform-defines.js

```diff
@@ -38,6 +38,7 @@ export function getPlatformShaderDefines(gl) {
 #define INTEL_GPU
 // Intel optimizes away the calculation necessary for emulated fp64
 #define LUMA_FP64_CODE_ELIMINATION_WORKAROUND 1
+#define LUMA_FP64_HIGH_BITS_OVERFLOW_WORKAROUND 1
```

Comment:
I think you need to define all flags in the "else" case as well. If we don't know what GPU it is, it could be that the GPU driver or browser is not implementing UNMASKED_RENDERER constant and not reporting a correct name.

You can add this as a comment to the else case.